tasks/task082_babi_t1_single_supporting_fact_question_generation.json

```diff
@@ -0,0 +1,6053 @@
+{
+    "Definition": "In this task, you will be given a passage consisting of set of facts. Among these facts, choose any one as a supporting fact, and rest of them will be irrelevant facts. The task is to create a question of form 'Where is <person_name>?' that is answerable from the supporting fact. Avoid creating questions that are unanswerable from all of the facts.",
```

Comment:
I think the sentence `Among these facts, choose any one as a supporting fact, and the rest of them will be irrelevant facts.` is not adding much to this task. If that's the case, let's drop it.

Reply:
Updated the definition in the new commit, let me know if that looks fine.